react/features/prejoin/components/native/Prejoin.tsx

```diff
@@ -149,14 +163,12 @@ const Prejoin: React.FC<IPrejoinProps> = ({ navigation }: IPrejoinProps) => {
                 }
                 <Button
                     accessibilityLabel = 'prejoin.joinMeeting'
-                    disabled = { joinButtonDisabled }
                     labelKey = 'prejoin.joinMeeting'
-                    onClick = { isJoining ? undefined : onJoin }
+                    onClick = { isJoining ? undefined : maybeJoin }
                     style = { styles.joinButton }
                     type = { PRIMARY } />
                 <Button
                     accessibilityLabel = 'prejoin.joinMeetingInLowBandwidthMode'
-                    disabled = { joinButtonDisabled }
```

Comment:
Why?

Reply:
We were only disabling this when the username was required and missing, but after what we've discussed, we agreed to leave the buttons enabled and prompt the display name on click.

Reply:
Ah right.